sgl-kernel/csrc/moe/moe_fused_gate.cu

```diff
@@ -57,6 +59,8 @@ __device__ void moe_fused_gate_impl(
     int64_t num_rows,
     int64_t topk_group,
```

Comment:
question: is it mathematically incorrect, for the shared expert fusion thing, if we do topk+=n_shared_experts_fusion, but keeping topk_group unchanged? (I have not digged into how it is computed, so just a quick question)